Changelog.md

```diff
@@ -1,5 +1,9 @@
 ### 0.8.21 (unreleased)
 
+Important Bugfixes:
```

Comment:
We decided *not* to classify these as important bugs.

Reply:
Changed to a normal bugfix.